src/audio/pulseaudio.rs

```diff
@@ -362,10 +363,15 @@ impl PulseAudioSoundDevice {
             }
         };
 
+        // Since we're adjusting the volume unmute
+        self.set_muted(false)?;
```

Comment:
I do not think this is the correct thing to do: If the user wishes to unmute on change, they could simply call set_muted(false) themselves. With this, you cannot avoid it.

I personally adjust my volume before I unmute to not accidentally burst my eardrums.

Reply:
Yeah, that's the other option and pacmd does it like that. I'll need to expose the mute/unmute as well so I can chain two commands in one bindsym like I do with pacmd.